### PR TITLE
FOGL-1599

### PR DESCRIPTION
**Pull request by AmandeepArora**
The get_plugin_info utility would be placed in "extras/C" directory in "sudo make install" version.
And at "cmake_build/C/plugins/common/get_plugin_info" in FogLAMP code tree after running make.

### CODE REVIEW

File: C/plugins/common/get_plugin_info.cpp [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/*
```
Comment (MarkRiddoch) [inline C/plugins/common/get_plugin_info.cpp]: This is in the wrong place - it is not common to plugins, it is a utility. Files in plugins/common is meant to be shared between multiple plugins.

Please add a plugins/utility directory for this.

Comment (AmandeepArora) [reply C/plugins/common/get_plugin_info.cpp]: ok sure.

**Comment by AmandeepArora** [comment]
@MarkRiddoch :  I made the required changes, please review.

**Comment by MarkRiddoch** [comment]
In the make rule we should copy the binary to FOGLAMP_ROOT/plugins/south for the plugins and FOGLAMP_ROOT/utils for the utility.

**Pull request merged**